dashboard/config/libraries/PoemBot.interpreted.js

```diff
@@ -0,0 +1,71 @@
+function poemBotDraw() {
+  if (isVisible) {
+    drawPoem();
+  }
+}
+
+function drawPoem() {
+  var outerMargin = 50;
+  var defaultLineHeight = 50;
+  var defaultFontSize = 25;
+  var cursor = outerMargin;
+  fill('black');
+  noStroke();
+  textSize(defaultFontSize);
+  textAlign(CENTER);
+  if (poem.title) {
+    textSize(defaultFontSize * 2);
+    text(poem.title, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += defaultLineHeight;
+  }
+  if (poem.author) {
+    cursor -= defaultLineHeight / 2;
+    textSize(16);
+    text("by: " + poem.author, 200, cursor);
```

Comment:
Do we need to think about non-en loc issues at some point?

Reply:
Yes, FND has it on their backlog to help with i18n in July